modules/core/test/test_arithm.cpp

```diff
@@ -2128,6 +2129,80 @@ TEST(Core_minMaxIdx, regression_9207_1)
 }
 
 
+class TransposeND : public testing::TestWithParam< tuple<std::vector<int>, perf::MatType> >
+{
+public:
+    std::vector<int> m_shape;
+    int m_type;
+
+    void SetUp()
+    {
+        std::tie(m_shape, m_type) = GetParam();
+    }
+};
+
+
+TEST_P(TransposeND, basic)
+{
+    setRNGSeed(42);
```

Comment:
> setRNGSeed(42);

should not be used in tests.
Tests has own seed through command line parameter.